emergency_behaviours/scripts/nogo_node_stop.py

```diff
@@ -118,7 +118,7 @@ def timer_callback(self):
             self.set_emergency_stop()
             if not self.nogo_pre_active :
                 self.update_service_list()
-                self.set_free_run(True)
+                self.set_free_run(False)
                 self.start_stop_scheduler(False)
```

Comment:
no close to the stairs free run is not active, it just look like that because my logic was inverted, I'll fix it so it is more readable